INSTALL.md

```diff
@@ -69,6 +69,24 @@ at [localhost:1081](http://localhost:1081).
 - Keep an eye on console in case of compiling errors.
 - [Read more](https://github.com/Trustroots/trustroots/wiki/Development)
 
+## Enable collecting statistics
+
+* 1. [Install InfluxDB](https://docs.influxdata.com/influxdb/latest/introduction/installation/) 0.9+ and run it (type `influxd`) at default port (8086)
```

Comment:
Hm, so actually now looking into this again, I see that I configured influxdb to be optional. So in development, if you just leave influxdb config off (and by default it is off), sending points will just fail silently.

So as long as developer doesn't need to directly work with influxdb, they can just ignore it.